invokeai/app/services/session_queue/session_queue_sqlite.py

```diff
@@ -133,8 +134,6 @@ async def enqueue_batch(self, queue_id: str, batch: Batch, prepend: bool) -> Enq
                     """,
                     values_to_insert,
                 )
-            with self._conn:
```

Comment:
Why do we implicitly commit here?

Reply:
This might be the core fix to implement...

Reply:
Not sure I understand how it would fix the issue... But it wouldn't hurt anything to move the select up into the first context.